src/customers/customers.service.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  BadRequestException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Customer } from './customers.entity';
+import { CreateCustomerDto } from '../dto/create-customer.dto';
+import { UpdateCustomerDto } from '../dto/update-customer.dto';
+
+@Injectable()
+export class CustomersService {
+  constructor(
+    @InjectRepository(Customer)
+    private readonly customersRepository: Repository<Customer>,
+  ) {}
```

Comment:
Maybe let's try to keep it without the shortcut for assigning the constructors arguments to properties, and aim for more "traditional" 😄 approach which is a bit longer but easier to read and understand
```suggestion
  private readonly customersRepository: Repository<Customer>
  
  constructor(
    @InjectRepository(Customer)
    customersRepository,
  ) {
    this.customersRepository = customersRepository
  }
```